SPEC.md

```diff
@@ -30,20 +30,20 @@ Type text.
 
 ##	Scope (mandatory)
 
-Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
 data structures for artifacts.  Including but not limited to:
```

Comment:
Remove EOL between "related" and "data"?

Reply:
This was semi intentional.

When rendered as markdown, the EOL will be disappear: markdown considers it a single paragraph.
When editing in an editor... it makes the line a reasonable length.